lib/rack/attack/configuration.rb

```diff
@@ -19,7 +19,7 @@ class Configuration
         end
       end
 
-      attr_reader :safelists, :blocklists, :throttles, :anonymous_blocklists, :anonymous_safelists
+      attr_reader :safelists, :blocklists, :throttles,  :tracks, :anonymous_blocklists, :anonymous_safelists
```

Comment:
```suggestion
      attr_reader :safelists, :blocklists, :throttles, :tracks, :anonymous_blocklists, :anonymous_safelists
```

there is an extra space there